test/performance/performance-tests-mako.sh

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
```

Comment:
Should we rename the script name?

Reply:
https://github.com/knative/serving/pull/14289#discussion_r1314464932